lib/good_job.rb

```diff
@@ -59,6 +59,14 @@ module GoodJob
   #   @return [Boolean, nil]
   mattr_accessor :retry_on_unhandled_error, default: false
 
+  # @!attribute [rw] cron_enabled
+  #   @!scope class
+  #   Use this to know if cron is enabled for the process (default: +false+).
+  #   If +true+, it means cron is enabled.
+  #   If +false+, it means cron disabled.
+  #   @return [Boolean, nil]
+  mattr_accessor :cron_enabled, default: false
```

Comment:
I'm trying to move away from these global accessors.

Take a look at `GoodJob::Configuration` and see if you can instantiate one of those instead to get the configuration.

Reply:
Should we directly add new key in process's current_state without creating this global accessor, like this -> `cron_enabled: Rails.application.config.good_job[:enable_cron]`

Reply:
I'm working on #681, which should allow us to use `GoodJob.configuration.enable_cron?`. I'll let you know when it gets merged.

Reply:
It's been merged. If you merge main or rebase, you can use `GoodJob.configuration`

Reply:
Okay, This made things easy.